c_emulator/rvfi_dii.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include "sail.h"
+
+typedef enum {
+  RVFI_prestep_continue,  // continue loop
+  RVFI_prestep_eof,       // Got EOF, delete rvfi and return
+  RVFI_prestep_end_trace, // Got EndOfTrace
+  RVFI_prestep_ok,        // Ready for step
+} rvfi_prestep_t;
+
+typedef void (*packet_reader_fn)(lbits *rop, unit);
+
+class rvfi_handler {
+  public:
+  rvfi_handler(int port);
+
+  int setup_socket(bool config_print);
+  uint64_t get_entry();
+  void send_trace(bool config_print);
+  rvfi_prestep_t pre_step(bool config_print);
+
+  private:
+  void get_and_send_packet(packet_reader_fn reader, bool config_print);
```

Comment:
```suggestion
public:
  rvfi_handler(int port);

  int setup_socket(bool config_print);
  uint64_t get_entry();
  void send_trace(bool config_print);
  rvfi_prestep_t pre_step(bool config_print);

private:
  void get_and_send_packet(packet_reader_fn reader, bool config_print);
```

The clang format configuration is doing some weird indentation here. I wonder why the access modifiers are indented.